minecraft_servers/voidroleplay/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "server_name": "voidroleplay",
+  "nice_name": "VoidRoleplay",
+  "direct_ip": "voidroleplay.de",
+  "server_wildcards": [
+    "%.voidroleplay.de"

```

Comment:
Not a real wildcard
https://i.imgur.com/9Kel6GY.png

Reply:
Thank you for noticing 👍, and sorry for my mistakes. The changes are made.